npm/angular/src/mount.ts

```diff
@@ -278,21 +278,22 @@ function setupComponent<T extends { ngOnChanges? (changes: SimpleChanges): void
  * @param {Type<T> | string} component Angular component being mounted or its template
  * @param {MountConfig<T>} config configuration used to configure the TestBed
  * @example
- * import { HelloWorldComponent } from 'hello-world/hello-world.component'
+ * import { mount } from '@cypress/angular'
+ * import { StepperComponent } from './stepper.component'
  * import { MyService } from 'services/my.service'
  * import { SharedModule } from 'shared/shared.module';
- * import { mount } from '@cypress/angular'
- * it('can mount', () => {
- *  mount(HelloWorldComponent, {
+ * it('mounts', () => {
+ *  mount(StepperComponent, {
  *    providers: [MyService],
  *    imports: [SharedModule]
  *  })
- *  cy.get('h1').contains('Hello World')
+ *  cy.get('[data-cy=increment]').click()
+ *  cy.get('[data-cy=counter]').should('have.text', '1')
  * })
  *
- * or
+ * // or
  *
- * it('can mount with template', () => {
+ * it('mounts with template', () => {
  *  mount('<app-hello-world></app-hello-world>', {
```

Comment:
In the '@returns', we can omit the type (its inferred from TS) and provide a brief description of the returns